test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -455,10 +458,10 @@ TEST(cmod_utilityrate5_eqns, Test_Residential_TOU_Rates_net_billing_carryover_ja
     ssc_data_set_number(data, "ur_nm_credit_month", 0); // January
 
     int analysis_period = 25;
-    ssc_data_set_number(data, "system_use_lifetime_output", 1);
+    ssc_data_set_number(data, "system_use_lifetime_output", 0);
     ssc_data_set_number(data, "analysis_period", analysis_period);
     set_array(data, "load", load_profile_path, 8760);
-    set_array(data, "gen", gen_path, 8760 * analysis_period);
+    set_array(data, "gen", one_year_gen_path, 8760);
```

Comment:
This needed to change to a non-lifetime simulation so that it was easier to match with Excel